jib-core/scripts/prepare_release.sh

```diff
@@ -4,39 +4,43 @@
 set -e
 
 Colorize() {
-	echo "$(tput setff $2)$1$(tput sgr0)"
+  echo "$(tput setff $2)$1$(tput sgr0)"
 }
 
 EchoRed() {
-	echo "$(tput setaf 1; tput bold)$1$(tput sgr0)"
+  echo "$(tput setaf 1; tput bold)$1$(tput sgr0)"
 }
 EchoGreen() {
-	echo "$(tput setaf 2; tput bold)$1$(tput sgr0)"
+  echo "$(tput setaf 2; tput bold)$1$(tput sgr0)"
 }
 
 Die() {
-	EchoRed "$1"
-	exit 1
+  EchoRed "$1"
+  exit 1
 }
 
 DieUsage() {
-    Die "Usage: ./scripts/prepare_release.sh <release version>"
+  Die "Usage: ./prepare_release.sh <release version> [<post-release-version>]"
 }
 
 # Usage: CheckVersion <version>
 CheckVersion() {
-    [[ $1 =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] || Die "Version not in ###.###.### format."
+  [[ $1 =~ ^[0-9]+\.[0-9]+\.[0-9]+(-[0-9A-Za-z]+)?$ ]] || Die "Version: $1 not in ###.###.###[-XXX] format."
 }
 
-[ $# -ne 2 ] || DieUsage
+[ $# -ne 2 ] || [ $# -ne 3 ] || DieUsage
 
 EchoGreen '===== RELEASE SETUP SCRIPT ====='
 
 VERSION=$1
 CheckVersion ${VERSION}
+if [ $2 ]; then
```

Comment:
It's good to wrap it in quotes `"$2"`, since otherwise if `$2` is undefined, the `if` syntax becomes broken as `if [  ]; then`.  Maybe `if [ -n "$2" ]` (testing if length is non-zero).